R/proj.R

```diff
@@ -0,0 +1,249 @@
+#' Calculate Projective Distributions, Averages, and Contrasts for a Summary Statistic
+#'
+#' @description
+#' The *projective distribution* of a district-level summary statistic (McCartan
+#' 2024) is the distribution of values of that statistic across a set of plans
+#' for the district each precinct belongs to. The *projective average* of a
+#' statistic is the average value of the projective distribution in each
+#' precinct. A *projective contrast* is the difference between the projective
+#' average for a single plan and the projective average for an ensemble of
+#' sampled plans.
+#'
+#' It is very important to properly account for variation in the projective
+#' distribution when looking at projective contrasts. The `pfdr` argument to
+#' `proj_contr()` will calculate q-values for each precinct that can be used to
+#' control the positive false discovery rate (pFDR) to avoid being misled by
+#' this variation. See [redist.plot.contr_fdr()] for a way to automatically
+#' plot projective contrasts with this false discovery rate control.
+#'
+#' @param plans A [redist_plans] object.
+#' @param x A district-level summary statistic calculated from the `plans`
+#'   object. Tidy-evaluated in `plans`.
+#' @param draws which draws/samples to include in the projective distribution.
+#'   `NULL` will include all draws, including reference plans. The special value
+#'   `NA` will include all sampled (non-reference) draws. An integer, logical,
+#'   or character vector indicating specific draws may also be provided.
+#'
+#' @references
+#' McCartan, C. (2024). Projective Averages for Summarizing Redistricting
+#' Ensembles. *arXiv preprint*. Available at \url{https://arxiv.org/abs/}.
```

Comment:
Can this be markdown or latex, consistently? Is there a reason to use `\url{}` here that I'm missing?